x-pack/plugins/ml/public/application/components/ml_inference/components/test_pipeline.tsx

```diff
@@ -147,27 +219,88 @@ export const TestPipeline: FC<Props> = memo(({ state, sourceIndex }) => {
         <EuiFlexItem>
           <EuiText color="subdued" size="s">
             <p>
-              <strong>
-                {i18n.translate(
-                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.optionalCallout',
-                  { defaultMessage: 'This is an optional step.' }
-                )}
-              </strong>
-              &nbsp;
-              <FormattedMessage
-                id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.description"
-                defaultMessage="Run a simulation of the pipeline to confirm it produces the anticipated results."
-              />{' '}
-              {state.targetField && (
+              {mode === TEST_PIPELINE_MODE.STEP ? (
+                <>
+                  <strong>
+                    {i18n.translate(
+                      'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.optionalCallout',
+                      { defaultMessage: 'This is an optional step.' }
+                    )}
+                  </strong>
+                  &nbsp;
+                </>
+              ) : null}
+              <>
                 <FormattedMessage
-                  id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.targetFieldHint"
-                  defaultMessage="Check for the target field {targetField} for the prediction in the Result tab."
-                  values={{ targetField: <EuiCode>{state.targetField}</EuiCode> }}
+                  id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.description"
+                  defaultMessage="Run a simulation of the pipeline to confirm it produces the anticipated results. {simulatePipelineDocsLink}"
+                  values={{
+                    simulatePipelineDocsLink: (
+                      <EuiLink external target="_blank" href={links.apis.simulatePipeline}>
+                        Learn more.
+                      </EuiLink>
+                    ),
+                  }}
                 />
+                <br />
+              </>
+              {state.targetField && (
+                <>
```

Comment:
Nit - I think this could do with a couple of `EuiSpacer`s (`m`) - one between the `Test the pipeline results` title and the text, and another after the 'Learn more' link and the next two lines.